lib/src/clojure_lsp/parser.clj

```diff
@@ -11,7 +11,7 @@
 #_{:clj-kondo/ignore [:clojure-lsp/unused-public-var]}
 (defmacro zspy [loc]
   `(do
-     (log/warn '~loc (pr-str (z/sexpr ~loc)))
+     (taoensso.timbre/warn '~loc (pr-str (z/sexpr ~loc)))
      ~loc))
```

Comment:
I wonder if we really need to add timbre to test alias just for this function, I don't use this at all as it just replace a str or a expr

Reply:
The reason to add it to the test alias is so you can actually log through koacha, not because of this macro.